draft-ietf-httpbis-client-hints.md

```diff
@@ -216,20 +216,14 @@ Implementers SHOULD support Client Hints opt-in mechanisms and MUST clear persis
 
 
 ## Deployment and Security Risks
-Adding new request headers forces us to think about two aspects:
+Deployment of new request headers requires several considerations:
 
-  - Will user-agent emitted values impact current servers which could be using these headers for a different purpose?
-  - Can malicious content impact servers on the user's network by crafting such request headers with malicious header values?
+  - Potential conflicts due to existing use of field name
+  - Properties of the data communicated in field value
 
-The former concern is a deployment risk, slightly mitigated by the opt-in mechanisms of Client Hints.
+In order to avoid conflicts and provide some guaranties regarding field values sent by user agents, specifications and features relying on Client Hints MUST use the "Sec-" prefix for request header names. User agents reserve this namespace to headers that can only be emitted by the browser. Therefore, they can guarantee and enforce the type and format of the data communicated in the header value fields. Further, specifications and features relying on Client Hints are encouraged to include "CH-" in the header name, in order to make it easier to distinguish Client Hint request headers from others.
```

Comment:
So, the first half of your recommendation, the "Sec-" prefix, is immediately explained as to why such a requirement exists.

However, the second half, "CH-", does not explain _why_ it's useful to distinguish Client Hint request headers from others.

**BEGIN**
Specifications and features relying on Client Hints MUST prefix request header names with a "Sec-" prefix. User agents, such as web browsers, ensure such prefixed headers may only be emitted by the browser, and application content, such as scripts, are forbidden from emitting such headers {{FETCH}}. This provides assurance that, when transmitted by user agents, that both the type and format of data communicated in such request headers are enforced by the user agent.

Additionally, specifications and features relying on Client Hints are RECOMMENDED to include the "CH-" in the header name, for example, in the form `Sec-CH-Foo` for a hypothetical Client Hint `Foo`. Client Hints provide additional information about a user agent [1] that previously may only have been accessible to scripts or content [2], and not necessarily transmitted in requests. Servers that wish to minimize the data collected in intermediate steps may wish to filter out all unrecognized or unprocessed "CH-" values from its records or intermediate steps, and a consistent use of "CH-" helps facilitate filtering new and unrecognized Client Hint Values.
**END**

[1] I'm not sure if this is strictly about UAs?
[2] I'm not entirely happy with that wording, but *shrug*

However, from our IETF 105 conversation, @mnot aptly pointed out that such concerns arguably apply to any/all new headers, and so perhaps the 'CH-' infix may be entirely unnecessary, because it's a generalized problem. The language I offer above attempts to provide explanation/justification for the RECOMMENDation, but I could also see with doing away with 'CH-' entirely.